workflow-templates/im-build-dotnet-ci.yml

```diff
@@ -592,12 +596,13 @@ jobs:
   finish-build:
     if: always() && needs.setup-build-workflow.outputs.CONTINUE_WORKFLOW == 'true'
     needs: [dotnet-test, jest, build-deployment-artifacts]
-    uses:  im-practices/.github/.github/workflows/im-reusable-finish-build-workflow.yml@v2
+    uses: im-practices/.github/.github/workflows/im-reusable-finish-build-workflow.yml@v2
     with:
       next-version: ${{ needs.build-deployment-artifacts.outputs.NEXT_VERSION }}
       title-of-teams-post: '<project-name> CI Build' # TODO: Replace <project-name>
       is-merge-to-main: ${{ needs.setup-build-workflow.outputs.IS_MERGE_TO_MAIN }}
-      additional-conclusions: | # TODO: Determine if there are any additional step outcomes/outputs you want considered or remove some items if they do not apply (like jest)
+      additional-conclusions:
+        | # TODO: Determine if there are any additional step outcomes/outputs you want considered or remove some items if they do not apply (like jest)
```

Comment:
Can you have the pipe on the next line?

Reply:
Good question.  My formatter helpfully did that for me...I put it back once but I guess I saved it again and it reformatted.  I'll fix that one again.